README.md

```diff
@@ -76,6 +76,19 @@ MIDDLEWARE = [
 ]
 ```
 
+## 2FA with OTP
+
+  You can require users to enter an OTP (One Time Password) along with their email to get the magic link.
+ - What is a OTP ? It is a short code (6 to 8 digits), usually generated by a dedicated app on a user's device (smartphone, hardware). See https://en.wikipedia.org/wiki/Time-based_One-time_Password_algorithm
+ - We use the [django-otp](https://django-otp-official.readthedocs.io/en/stable/) package. You must have it installed and configured in your project
+ - How to enable ?
+   1. install `django_otp` and run the migrations in your project (prerequisite)
```

Comment:
nstall django_otp package, add the necessary packages to INSTALLED_APPS (see https://django-otp-official.readthedocs.io/en/stable/overview.html#installation) and run the migrations in your project (prerequisite)